docs/advanced-guides/testbed.md

```diff
@@ -76,6 +76,7 @@ The OSISM Testbed requires at least the following project quota when using the d
 
 ### Software
 
+* currently only Ubuntu 22.04 is supported for the manager base distribution, this is caught by a precheck
```

Comment:
This is confusing at this point as it is about the necessary tools on the workstation. On the workstation, it doesn't matter which operating system it is.

Reply:
It makes sense to mention this in the description of the variable `image`.